packages/core/src/shared/languageServer/lspResolver.ts

```diff
@@ -0,0 +1,352 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import fs from '../fs/fs'
+import { ToolkitError } from '../errors'
+import * as semver from 'semver'
+import * as path from 'path'
+import { FileType } from 'vscode'
+import AdmZip from 'adm-zip'
```

Comment:
We have a zip.js-based ZipStream module now. We should avoid adm-zip , because we want to remove it.